frontend/src/components/HomeComponents/NavList.js

```diff
@@ -29,6 +29,9 @@ export default function NavList() {
           </li>
           <li>
             <NavLink
+                      className={({ isActive }) => 
```

Comment:
You don't need to add this isActive styling to all the components since the navbar isn't visible in other sections or when scrolled down.

Also, instead of using className as hoverEffect, try to use a more generic name like navLink
or something similar.